api/external/envoy/service/discovery/v2/ads.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
```

Comment:
Needed in Gloo https://github.com/solo-io/gloo/blob/master/projects/gloo/pkg/xds/envoy_server_v2.go#L50